src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {
+      person.age
+        ? <p className="Person__age">{`I am ${person.age}`}</p>
+        : ''
+    }
+    {
+      person.isMarried === true
```

Comment:
```suggestion
      person.isMarried
```

not needed because it's converted to boolean